src/morphic.js

```diff
@@ -5357,73 +5357,154 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
         this.target.setAlignmentToLeft();
     }
     this.gotoSlot(this.slot);
-    this.initializeClipboardHandler();
+    this.initializeTextarea();
 };
 
-CursorMorph.prototype.initializeClipboardHandler = function () {
-    // Add hidden text box for copying and pasting
-    var myself = this,
-        wrrld = this.target.world();
+CursorMorph.prototype.initializeTextarea = function () {
+    var myself = this;
 
-    this.clipboardHandler = document.createElement('textarea');
-    this.clipboardHandler.style.position = 'absolute';
-    this.clipboardHandler.style.top = window.outerHeight;
-    this.clipboardHandler.style.right = '101%'; // placed just out of view
+    this.textarea = document.createElement('textarea');
+    this.textarea.style.zIndex = -1;
```

Comment:
Minor thing:
For the style elements that are static, I think it's clearer to write

```suggestion
    this.textarea.setAttribute('style', 'z-index: -1; position: absolute; overflow: hidden')'; 
```

which keeps it all together, then `wrap` and `autofocus` can be next to each other (since they are both attributes specific to the element.)

Reply:
Thanks for your suggestion, I'll change it in next PR.

I've never thought this way: "this.textarea.style.zIndex=1" is changing the "style" attribute of the element.